tests/rest/client/test_typing.py

```diff
@@ -59,7 +59,7 @@ def test_set_typing(self) -> None:
             self.event_source.get_new_events(
                 user=UserID.from_string(self.user_id),
                 from_key=0,
-                limit=None,
+                limit=0,
```

Comment:
Oh, so `0` means `no limit`?

(Confusing---that's what I thought `limit=None` meant)

Reply:
This parameter is unused in the typing event source. 😢 

`EventSource` sub-classes are a mess.

Reply:
I added a comment saying this is unused.